packages/ember/index.js

```diff
@@ -16,32 +16,15 @@ module.exports = {
     },
   },
 
-  getAddonConfig(app) {
-    let config = {};
-    try {
-      config = require(app.options.configPath)(app.env);
-    } catch(_) {
-      // Config not found
-    }
-    return config['@sentry/ember'] || {};
-  },
-
-  included() {
-    this._super.included.apply(this, arguments);
-    const app = this._findHost(this);
-    if (!('@embroider/core' in app.dependencies())) {
-      const addonConfig = this.getAddonConfig(app);
-      const options = Object.assign({}, addonConfig);
-      this.options['@embroider/macros'].setOwnConfig.sentryConfig = options;
-    }
+  config(_, appConfig) {
+    const addonConfig = appConfig['@sentry/ember'];
+    const options = Object.assign({}, addonConfig);
+    this.options['@embroider/macros'].setOwnConfig.sentryConfig = options;
```

Comment:
```suggestion
    this.options['@embroider/macros'].setOwnConfig.sentryConfig = { ...addonConfig };
```

Reply:
Unless it's fine to reuse the same object, then even deconstruction is unnecessary.